src/Dialect/ONNX/ONNXOps/OpHelper.cpp

```diff
@@ -317,6 +317,24 @@ ONNXConstantOp getONNXConstantOp(Value value) {
   return dyn_cast_or_null<ONNXConstantOp>(value.getDefiningOp());
 }
 
+bool getI64ValuesFromSmallONNXConstantOp(
```

Comment:
Just to confirm: does it work with DisposableElementsAttrs? I guess the issue is the value in ONNXConstantOp can be DisposableElementsAttrs or DenseElementsAttrs. That's why I convert to DisposableElementsAttrs to DenseElementsAttrs instead of reading values directly from DisposableElementsAttrs, and it's expensive.

Reply:
ElementsAttrs works well. Thanks!